### PR TITLE
Fixes first run wizard check;Makes network create mandatory step in f…

### PR DESCRIPTION
**Pull request by vishal-j**
…irst run wizard

### CODE REVIEW

File: app/login/loginctrl.ts [hunk 1]
```diff
@@ -51,20 +51,23 @@ export class LoginComponent implements OnInit{
                 if(result){
                     this.showServerError = false;
                     this.crudHelperService.stopLoader(this);
+                    this.firstRunService.setFirstRun()
+                        .then(isFirstRun => {
+                            if (isFirstRun) {
```
Comment (shampur) [inline app/login/loginctrl.ts]: I think we should stop the loader after the firstrun check.

File: app/app.module.ts [hunk 2]
```diff
@@ -39,6 +39,7 @@ import { ApplicationGroupsModule } from "./applicationgroups/applicationgroups.m
 import   appRoutes  from "./app.routes";
```
Comment (shampur) [inline app/app.module.ts]: This is not related to the code, I thought of adding a favicon to the UI.
We can add this favicon in index.html head section: 
<link rel="icon"
          type="image/png"
          href="images/contiv_36x36.png">

File: app/login/loginctrl.ts (hunk 1 above)
Comment (vishal-j) [inline app/login/loginctrl.ts]: Fixed.

File: app/app.module.ts (hunk 2 above)
Comment (vishal-j) [inline app/app.module.ts]: Added favicon
